default.nix

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is there any way we can keep the .nix files out of the SDK?

Reply:
I'll remove it.